src/validators/v1/actions.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * name : validators/v1/actions.js
+ * author : Priyanka Pradeep
+ * Date : 14-Aug-2024
+ * Description : Validations of actions controller
+ */
+
+module.exports = {
+	update: (req) => {
+		req.checkParams('id')
+			.trim()
+			.optional()
+			.notEmpty()
+			.withMessage('id param is empty')
+			.isNumeric()
+			.withMessage('id param is invalid, must be an integer')
+
+		req.checkBody('code')
```

Comment:
@priyanka-TL - I agree with @adithyadinesh0412 only what is required or needs to be updated should be passed in update call, in this code might not be required.

Reply:
@aks30 update API is for create and update action. I made the changes, if id is there code and description i made as optional when id is there